viper.go

```diff
@@ -148,14 +149,20 @@ func (b *Bundle) provideViper(ctx context.Context, flagSet *pflag.FlagSet) (_ *v
 
 	var configFile string
 	if configFile, err = flagSet.GetString("config"); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("flag error: flag config not setup: %v", err)
```

Comment:
Better use error wrapping. Please change the fmt.Errorf first argument to to `unable to get config flag value  : %w`